sky/spot/constants.py

```diff
@@ -7,3 +7,6 @@
 SPOT_CONTROLLER_YAML_PREFIX = '~/.sky/spot_controller'
 
 SPOT_TASK_YAML_PREFIX = '~/.sky/spot_tasks'
+
+SPOT_WORKDIR_BUCKET_NAME = 'sky-spot-workdir-{username}-{hash}'
+SPOT_FILE_MOUNT_BUCKET_NAME = 'sky-spot-fm-{username}-{hash}'
```

Comment:
nits:
- `fm` -> `file-mounts` / `filemounts`?  As buckets are user-facing, this may be easier to spot.
- (for discussion) wdyt about starting to use `skypilot` for such new constants?

Reply:
Good point! Fixed. Thanks!